packages/web-react/src/components/Pill/__tests__/Pill.test.tsx

```diff
@@ -28,7 +31,18 @@ describe('Pill', () => {
     expect(element.textContent).toBe('3');
   });
 
-  it.each([['selected'], ['danger'], ['informative'], ['success'], ['warning']])('should render color %s', (color) => {
+  it.each([
```

Comment:
I think this should by covered by `actionColorPropsTest` and `emotionColorPropsTest`, but I see only one of them.